.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Playground Tools issues - wp-now, VS Code extension, utilities
+about: Did a tool not work the way you want it to? Let us know!
+title: Report an issue with the Playground Tools
+labels: bug
+assignees: ''
+
+---
+
+<!-- 
+Report issues with online Playground instances [here.](https://github.com/WordPress/wordpress-playground/issues)
+-->
+
+## How are you using the Playground?
+
+<!-- 
```

Comment:
And then we could remove this comment or make it a single line of text to save verticsl space